android/expoview/src/main/java/host/exp/exponent/ExpoUpdatesAppLoader.kt

```diff
@@ -413,10 +413,13 @@ class ExpoUpdatesAppLoader @JvmOverloads constructor(
       return headers
     }
 
+  private val isRunningOnEmulator: Boolean
+    get() = EmulatorUtilities.isRunningOnEmulator()
+
   private val clientEnvironment: String
     get() = if (Constants.isStandaloneApp()) {
       "STANDALONE"
-    } else if (Build.FINGERPRINT.contains("vbox") || Build.FINGERPRINT.contains("generic")) {
+    } else if (isRunningOnEmulator) {
```

Comment:
```suggestion
    } else if (EmulatorUtilities.isRunningOnEmulator()) {
```

maybe we could just use the `EmulatorUtilities.isRunningOnEmulator()` here and saving some java bytecode size.